.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should js formatters be in main directory?

Reply:
Prettier also formats markdown files, thus it is in the main directory.